cpu/kinetis/include/cpu_conf_kinetis.h

```diff
@@ -72,6 +72,11 @@ extern "C"
 #if !defined(OSC0) && defined(OSC)
 #define OSC0 OSC
 #endif
+#if !defined(SIM_SOPT2_LPUART0SRC_MASK) && defined(SIM_SOPT2_LPUARTSRC_MASK)
+#define SIM_SOPT2_LPUART0SRC_MASK SIM_SOPT2_LPUARTSRC_MASK
+#define SIM_SOPT2_LPUART0SRC_SHIFT SIM_SOPT2_LPUARTSRC_SHIFT
+#define SIM_SOPT2_LPUART0SRC SIM_SOPT2_LPUARTSRC
+#endif
```

Comment:
Can these defines be aligned? Just to see that they're actually defining a value.

Reply:
Will fix